test/v1/models/response/CreateAndAmendOtherDeductionsResponseSpec.scala

```diff
@@ -16,9 +16,10 @@
 
 package v1.models.response
 
+import api.models.hateoas
```

Comment:
Like `CreateAndAmendOtherDeductionsResponse` can we move this into a dedicated directory (see comment on the CreateAndAmendOtherDeductionsResponse file for more detail)